packages/grafana-ui/src/components/TransformersUI/OrganizeFieldsTransformerEditor.tsx

```diff
@@ -133,12 +138,15 @@ const DraggableFieldName: React.FC<DraggableFieldProps> = ({
                 surface="header"
                 onClick={() => onToggleVisibility(fieldName, visible)}
               />
-              <span className={styles.name}>{fieldName}</span>
+              <span className={styles.name} title={fieldName}>
+                {fieldName}
+              </span>
             </div>
             <Input
               className="flex-grow-1"
+              defaultValue={renamedFieldName || ''}
```

Comment:
this is confusing, is this not the same as fieldName ?

Reply:
No, the original field name is displayed on the left side as the label, in the input we are showing the name that was configured.

Reply:
still do not understand, should it not be setting value={renamedFieldName} then or is this an uncontrolled input?

Reply:
@torkelo mind the change from onChange to onBlur. From performance reason I don't want the view to update on every keystroke. Having `value` requires `onChange` which would have to be a noop.

Reply:
https://github.com/grafana/grafana/pull/23812/files/c2a7e4a851623b49eda73aa92b0daedc018d4c32#diff-6a04ae10d31ea5d1f589ea8ac37af893R149